### DISCUSSION TITLE
context_length_exceeded

### DISCUSSION THREAD

**Discussion by Ningyezi** [MuiseDestiny/zotero-gpt#162] [Q&A]
![Uploading 1684756969909.png…]()
在连续问了几个问题后，它就出现这个错误代码，可能是因为新添加的功能，上下文的缘故？

**Comment by Ningyezi** [comment]
error code: 

”context_length_exceeded：

invalid_request_error
This model's maximum context length is 4097 tokens. However, your messages resulted in 6059 tokens. Please reduce the message.”

但是我的prompt只有几个词语，不应该超数量了。

**Reply by Ningyezi** [reply]
@MuiseDestiny ask for help.

**Reply by MuiseDestiny** [reply]
run "/clear" to clear your history.

![image](https://github.com/MuiseDestiny/zotero-gpt/assets/51939531/1bf0a097-db1f-41aa-8bf8-bfb00f082b42)
